app/src/main/java/com/globant/example/mentorapp/mvp/base/BaseRecyclerViewAdapter.java

```diff
@@ -0,0 +1,16 @@
+package com.globant.example.mentorapp.mvp.base;
+
+/**
+ * Generic class to define basic RecycleView behavior
+ * Created by oscar.amezquita on 21/07/2017.
+ */
+
+public class BaseRecyclerViewAdapter {
```

Comment:
Do you really need to define this class, do you expect to implement onUserClick in many places?